spring-pulsar-spring-boot-autoconfigure/src/test/java/org/springframework/pulsar/autoconfigure/PulsarPropertiesTests.java

```diff
@@ -231,10 +236,15 @@ void adminProperties() {
 			bind(props);
 			Map<String, Object> adminProps = properties.buildAdminProperties();
 
-			// Verify that the props can be loaded in a ClientBuilder
-			assertThatNoException().isThrownBy(() -> PulsarAdmin.builder().loadConf(adminProps));
+			// Verify that the props can NOT be loaded directly via a ClientBuilder due to
+			// the
+			// unknown readTimeout and autoCertRefreshTime properties
+			assertThatRuntimeException().isThrownBy(() -> PulsarAdmin.builder().loadConf(adminProps)).havingCause()
```

Comment:
Once the duration are out of the picture, the CCD barfs on the unknown props `readTimeout` and `autoCertRefreshTime`. I chose to keep this negative test in here really for consistecy w/ the other tests and also info share to this fact that it can not be directly loaded w/o some intervention.